.eslintrc.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  root: true,
+  extends: ['@stacks/eslint-config', 'prettier'],
+  overrides: [],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    tsconfigRootDir: __dirname,
+    project: './tsconfig.json',
+    ecmaVersion: 2020,
+    sourceType: 'module',
+  },
+  ignorePatterns: ['*.config.js', 'config/*'],
+  plugins: ['@typescript-eslint', 'eslint-plugin-tsdoc', 'prettier'],
+  rules: {
+    'prettier/prettier': 'error',
+    '@typescript-eslint/no-inferrable-types': 'off',
+    '@typescript-eslint/camelcase': 'off',
+    '@typescript-eslint/no-empty-function': 'off',
+    '@typescript-eslint/no-use-before-define': ['error', 'nofunc'],
+    '@typescript-eslint/no-floating-promises': ['error', { ignoreVoid: true }],
+    'no-warning-comments': 'warn',
+    'tsdoc/syntax': 'error',
+    // TODO: Remove this when `any` abi type is fixed.
```

Comment:
Can some of the safer/stricter rules be enabled here? This comment shouldn't apply this this repo right?

Reply:
I gave this a try, but since we're using `any` aliased as `PgJsonb` to insert into those columns in postgres we won't be able to effectively get rid of these (plus the other required `any` types for some metadata property values as specified in another comment).